src/webgl/loading.js

```diff
@@ -106,20 +106,23 @@ p5.prototype.loadModel = function(path) {
   let normalize;
   let successCallback;
   let failureCallback;
-  let fileType;
+  let fileType = path.slice(-4);
   if (typeof arguments[1] === 'boolean') {
     normalize = arguments[1];
     successCallback = arguments[2];
     failureCallback = arguments[3];
-    fileType = arguments[4];
+    if (arguments[4] != null) {
```

Comment:
Can we do a `typeof arguments[4] != 'undefined'` here and on line 121 instead? It is more consistent with the style of the rest of the library.

Reply:
Sure, I added that.